gaek/ndb_json.py

```diff
@@ -128,8 +127,16 @@ def default(self, obj):
 
     obj_type = type(obj)
     # NDB Models return a repr to calls from type().
-    if obj_type not in NDB_TYPE_ENCODING and hasattr(obj, '__metaclass__'):
-      obj_type = obj.__metaclass__
+    if obj_type not in NDB_TYPE_ENCODING:
+      if hasattr(obj, '__metaclass__'):
+        obj_type = obj.__metaclass__
+      else:
+        # Try to encode subclasses of types
+        for ndb_type in NDB_TYPE_ENCODING.keys():
```

Comment:
Oh interesting-- I can definitely see how that could happen.  If we wanted to avoid that problem then really this code would need to find the type that was _the most specific_ subclass of the object.  I think that case would be sufficiently rare that this code would be fine, but if we decide differently we could always go mucking around with the class.mro()!